tests/behaviour/cube.feature

```diff
@@ -251,10 +380,179 @@ Feature: Cube!
-          {'csv_column_title': 'Value', 'structural_definition': {'data_type': 'decimal', 'unit': None}, 'csv_column_uri_template': None, 'uri_safe_identifier_override': None}, 
-          {'csv_column_title': 'Measure', 'structural_definition': {'measures': [{'label': 'Length', 'description': None, 'parent_measure_uri': None, 'source_uri': None, 'uri_safe_identifier_override': None, 'arbitrary_rdf': []}]}, 'csv_column_uri_template': None, 'uri_safe_identifier_override': None}, 
-          {'csv_column_title': 'Unit', 'structural_definition': {'units': [{'label': 'cm', 'description': None, 'source_uri': None, 'uri_safe_identifier_override': None, 'arbitrary_rdf': [], 'base_unit': None, 'base_unit_scaling_factor': None, 'qudt_quantity_kind_uri': None, 'si_base_unit_conversion_multiplier': None}]}, 'csv_column_uri_template': None, 'uri_safe_identifier_override': None}
-      ]
+    [
+      "Dimension",
+      "Value",
+      "Measure",
+      "Unit"
+    ]
+    """
+    Then The cube data should match
+    """
+    [
+      {
+        "Dimension":"a",
+        "Value":0,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"b",
+        "Value":1,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"c",
+        "Value":2,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"d",
+        "Value":3,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"e",
+        "Value":4,
+        "Measure":"length",
+        "Unit":"cm"
+      }
+    ]
+    """
+
+    Scenario: Output a cube when the code list is defined using code list config schema v1.0 and when there are references to concepts defined elsewhere.
```

Comment:
I think this one needs to be checking the code lists which have been written to file instead of the cube information itself. I wonder whether this is the right feature file for it to be part of.

Reply:
I have added a new unit test for testing the same_as field generated for the concepts defined elsewhere. See `test_code_list_config_with_concepts_defined_elsewhere` in `test_code_list_config.py`.